tests/test_meta_functions_against_registry.py

```diff
@@ -163,6 +187,43 @@ def test_get_underlying_coins(metaregistry, registries, max_pools, curve_api):
             assert actual_output == metaregistry_output
 
 
+def test_get_underlying_balances(metaregistry, registries, max_pools, curve_api):
+    for i, registry in enumerate(registries):
+        total_pools = registry.pool_count() if max_pools == 0 else max_pools
+        for pool_index in range(total_pools):
+            metaregistry_output = None
+            pool = registry.pool_list(pool_index)
+            metaregistry_borks = False
```

Comment:
Don't use "borks" when naming variables/functions. Make your variable naming clear ("error") or if it's for exception handling, just do exception handling instead of keeping booleans.